go.mod

```diff
@@ -10,13 +10,14 @@ require (
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
+	github.com/kubernetes-csi/external-snapshotter/client/v2 v2.0.0-00010101000000-000000000000
```

Comment:
"make test" could not pass.  Maybe because I've cut a v2.2.0-rc1.  Try update this to v2.2.0

Reply:
@xing-yang i have addressed the changes for vendor folder.Could you please take a look